src/Compilers/CSharp/Portable/Symbols/OverriddenOrHiddenMembersHelpers.cs

```diff
@@ -540,7 +565,8 @@ internal static OverriddenOrHiddenMembersResult MakeInterfaceOverriddenOrHiddenM
                             break;
 
                         default:
-                            if (exactMatchComparer.Equals(member, otherMember))
+                            if (otherMember.Equals(explicitOverride, TypeCompareKind.AllIgnoreOptions) ||
```

Comment:
>explicitOverride [](start = 51, length = 16)

Is it possible that we don't reach the ```explicitOverride``` because we reached some other "suitable" candidate?

Reply:
Yes, and that would indicate a binary compatibility scenario (overrides inserted into the hierarchy) in which that is the right thing to do. Tests for that scenario will appear in the next PR (#44025)

---
In reply to: [421737450](https://github.com/dotnet/roslyn/pull/43797#discussion_r421737450) [](ancestors = 421737450)